w3lib/encoding.py

```diff
@@ -22,14 +22,15 @@ def http_content_type_encoding(content_type):
 
 # regexp for parsing HTTP meta tags
 _TEMPLATE = r'''%s\s*=\s*["']?\s*%s\s*["']?'''
+_SKIP_ATTRS = r'''(?:\s+[\w-]+=(?:'[^']*'|"[^"]*"|[^'"\s]+))*?'''
```

Comment:
Hey @Digenis,
Sorry for a review taking so long!
Could you please reformat and comment this regex to make it easier to read?